whiteboard-challenge-03/lib/array_binary_search.js

```diff
@@ -1,32 +1,37 @@
 'use strict';
 
-var binarySearch = function(arr, val) {
-    var half = Math.floor(arr.length / 2);
+class ArrayBinarySearch {
 
-    if(arr.length === 1) {
-        if (arr[0] === val) {
+    binarySearch(arr, val) {
+        var half = Math.floor(arr.length / 2);
```

Comment:
"Without utilizing any of the built-in methods available to your language" <- Be careful about this